helm/charts/wepublish/values.yaml

```diff
@@ -5,13 +5,26 @@ deployment:
   env: {}
 
   api:
-    env: {}
+    env:
+      GOOGLE_APPLICATION_CREDENTIALS: /var/secrets/google/key.json
+      MEDIA_SERVER_URL: https://media
+      MEDIA_ADDRESS: media_server_address
+      MEDIA_PORT: "443"
+      DATABASE_URL: postgres://
```

Comment:
Why not use different place holders?

Reply:
You mean why put everything into the ENV?
Just ease of adding things